src/devices/Display/LedDriver/Ht16k33.cs

```diff
@@ -0,0 +1,213 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+using Iot.Device.Display.LedDriver.Enums;
```

Comment:
I'd collapse .Enums subnamespace and put stuff directly under LedDriver

Reply:
Just this one under LedDriver? Or all the others too?